packages/components/src/wordpress-logo/index.tsx

```diff
@@ -1,19 +1,46 @@
 import './style.scss';
 
-type Props = {
+type LogoProps = {
 	className?: string;
 	size?: number;
 };
 
-const WordPressLogo: React.FunctionComponent< Props > = ( {
+export const WordPressLogo: React.FunctionComponent< LogoProps > = ( {
 	className = 'wordpress-logo',
 	size = 72,
 } ) => {
 	return (
-		<svg className={ className } height={ size } width={ size } viewBox="0 0 72 72">
+		<svg
+			className={ className }
+			height={ size as number }
+			width={ size as number }
+			viewBox="0 0 72 72"
+		>
 			<path d="M36,0C16.1,0,0,16.1,0,36c0,19.9,16.1,36,36,36c19.9,0,36-16.2,36-36C72,16.1,55.8,0,36,0z M3.6,36 c0-4.7,1-9.1,2.8-13.2l15.4,42.3C11.1,59.9,3.6,48.8,3.6,36z M36,68.4c-3.2,0-6.2-0.5-9.1-1.3l9.7-28.2l9.9,27.3 c0.1,0.2,0.1,0.3,0.2,0.4C43.4,67.7,39.8,68.4,36,68.4z M40.5,20.8c1.9-0.1,3.7-0.3,3.7-0.3c1.7-0.2,1.5-2.8-0.2-2.7 c0,0-5.2,0.4-8.6,0.4c-3.2,0-8.5-0.4-8.5-0.4c-1.7-0.1-2,2.6-0.2,2.7c0,0,1.7,0.2,3.4,0.3l5,13.8L28,55.9L16.2,20.8 c2-0.1,3.7-0.3,3.7-0.3c1.7-0.2,1.5-2.8-0.2-2.7c0,0-5.2,0.4-8.6,0.4c-0.6,0-1.3,0-2.1,0C14.7,9.4,24.7,3.6,36,3.6 c8.4,0,16.1,3.2,21.9,8.5c-0.1,0-0.3,0-0.4,0c-3.2,0-5.4,2.8-5.4,5.7c0,2.7,1.5,4.9,3.2,7.6c1.2,2.2,2.7,4.9,2.7,8.9 c0,2.8-0.8,6.3-2.5,10.5l-3.2,10.8L40.5,20.8z M52.3,64l9.9-28.6c1.8-4.6,2.5-8.3,2.5-11.6c0-1.2-0.1-2.3-0.2-3.3 c2.5,4.6,4,9.9,4,15.5C68.4,47.9,61.9,58.4,52.3,64z" />
 		</svg>
 	);
 };
 
-export default WordPressLogo;
+interface WatermarkProps extends Omit< LogoProps, 'size' > {
+	size?: { width: number; height: number };
+}
+
+export const WordPressWordmark: React.FunctionComponent< WatermarkProps > = ( {
```

Comment:
Why isn't the `WordPressWordmark` in its own separate file? In an ideal world, we may want to bundle only one or the other and not add extra unused bytes.